### ISSUE TITLE
Make `scaling` more user friendly

### ISSUE THREAD

**Issue by gavinsimpson** [vegandevs/vegan#127]
I am forever forgetting which way round the scaling numbers go for "site"- or "species"-focussed scalings. The numerical concept for scalings is canonical because of Canoco (& related software) but even that has long-since dropped that convention in the main user interface.

Proposal is to allow informative, non-numeric, indicators for the type of scaling within *all* functions that support a `scaling` argument and where such a change makes sense.

Thoughts on the initial idea before we speak to specifics of how to implement and what to use for the scaling *strings*?

Places we need to change this are, at a minimum:

 * [x] R/biplot.rda.R
 * [x] R/plot.cca.R
 * [x] R/plot.prc.R
 * [x] R/points.cca.R
 * [x] R/predict.cca.R
 * [x] R/predict.rda.R
 * [x] R/scores.cca.R
 * [x] R/scores.rda.R
 * [x] R/summary.cca.R
 * [x] R/summary.prc.R
 * [x] R/text.cca.R
 * [x] R/tolerance.cca.R

**Comment by jarioksa** [comment]
I agree: this is a good idea. 

Strings are easy to find for non-negative values (for instance, `"none"`, `"sites"`, `"species"`, `"symmetric"`). However, the coding got overloaded with negative scores that have different meaning for `cca` and `rda` objects. How to name those?

**Comment by gavinsimpson** [comment]
To do this properly, I think we need to introduce a new argument and treat linear, unimodal, others(?) separately. For unimodal methods negatives are so-called "Hill's scaling" in Canoco. For linear methods Canoco divides by the variable standard deviation, which is also what we do is it not?

If so, `scores.cca` could have argument `hill = FALSE`, and new `scores.rda` could have `standardise = FALSE` to invoke the negative scores. We'd ignore these new args if `scaling` is supplied as a length one integer.

**Comment by gavinsimpson** [comment]
@jarioksa Any thoughts on suggested new arguments `hill = FALSE` and `standardise = FALSE` (or `scale = FALSE`) for `scores.cca` and `scores.rda` respectively?

**Comment by jarioksa** [comment]
Yes, I think it makes sense to add those new keywords. The main keyword is `scaling` and `scale`  could be confusing to modify its meaning in `scores.rda`. Probably something else is needed, although `standardize` also sounds confusing. For `scores.cca` keyword `hill` is fine (but I don't suggest using some person name for `scores.rda`). Alternatively, `dcalike` for `scores.cca` and `rlike` or `corrlike` or something similar could work for `scores.cca` and `scores.rda`. However, I'm not particular about these names: the main design is OK, and similar to one I was thinking myself.

The `scores()` functions are used all over **vegan**, and it may need some work to check that new parameters work in all cases. Currently there are
```sh
vegan$ grep -l "scores(" R/*.R | wc -l
47
```
functions that use `scores()`. However, maintaining compatibility with old numeric `scaling` will save us from a lot of trouble.

**Comment by gavinsimpson** [comment]
Re the argument name for `scores.rda`, how about `correlation`? It is not abbreviated so you don't have to remember if it is `corlike` or `corrlike`, and the help file can explain that these are correlation-like scores.

Unless other objects, I will implement a solution using `hill = FALSE` and `correlation = FALSE` for `scores.cca` and `scores.rda` respectively.

I plan to write a single utility function that will do the correct thing and return the appropriate numeric score for arguments `scaling` and `hill` or `correlation` and drop that into various places we use this. However, it is more likely that we only need to modify those places where we currently have `scaling`:

```bash
vegan]$ grep -l "scaling =" R/*.R | wc -l
13
```

bc otherwise we were calling `scores()` assuming `...` would be passed on to it. The real value of places to change may be somewhere between 47 and 13 however.

**Comment by jarioksa** [comment]
Closed with PR #137
